app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="w-100 mt-5">
+<div class="d-flex justify-content-center flex-column">
+<h2 class="display-3 mb-5">Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions mt-3">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] On submitting the login form it's not possible to tell if the login was successful or not. This is because there is no confirmation message and the user is not redirected. Kindly redirect the user and display a success message on the register.
- [ ] Kindly display error messages in case the user tries to submit a form with empty fields.
- [ ] Kindly adjust the login and the register page to match the given wireframe:
![Screenshot from 2022-02-09 12-06-05](https://user-images.githubusercontent.com/20245164/153162115-5290f331-ccd0-4ac7-96d3-67910a531b22.png)
See how the form fields and the submit button are stretched??

Reply:
This issue is well resolved! ✔️